string vehicle_id
float32 position_lat
float32 position_long
bool isRoadLeader